acme/setup.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+install_requires = [
+    'argparse',
+    # load_pem_private/public_key (>=0.6)
+    # rsa_recover_prime_factors (>=0.8)
+    'cryptography>=0.8',
+    'mock',
```

Comment:
I have a patch ready, where conditional version tag is applied for Python 2.6. This way `debian` branch doesn't have to diverge from `master`. Same for all other `setup.py` scripts.